es6-promises/main.js

```diff
@@ -0,0 +1,11 @@
+import takeAChance from './take-a-chance.js';
+
+const user = takeAChance('Justin');
+
+user.then((value) => {
+  console.log(value);
+},
+(reason) => {
+  console.error(reason.message);
+}
+);
```

Comment:
This works, but the preferred approach is to use `.catch` instead of both fulfill and reject handlers in `.then`.